app/routes/home.js

```diff
@@ -6,13 +6,15 @@ export default PublicRoute.extend({
     let applicationController = this.controllerFor("application");
     controller.set("cartLength", applicationController.get("cartLength"));
     controller.set("hasCartItems", applicationController.get("hasCartItems"));
+    applicationController.set("pageTitle", this.get("i18n").t("browse.title"));
     applicationController.set("isHomePage", true);
   },
 
   resetController: function(controller, isExiting) {
     this._super.apply(this, arguments);
     if (isExiting) {
-      this.controllerFor("application").set("isHomePage", false);
+      let controller = this.controllerFor("application");
+      controller.set("isHomePage", false);
```

Comment:
this is no longer needed, the `isHomePage` property is now a computed property in the application controller : 

```javascript
isHomePage: Ember.computed("currentPath", function() {
    return this.get("currentPath") === "home";
  })
```

Reply:
Yeah, I'm removing it.